env/README.md

```diff
@@ -7,41 +7,196 @@ compilation of the CROSSCON hypervisor.
 
 ## How to build the container
 
-### Build the docker image
+Run this command:
 
-Run the `./build.sh` to build the image. The resulting image will have
-the tag: `crosscon_hv`.
+```bash
+docker build -t crosscon_hv .
+```
 
-This will also produce the `build.log` log file, which is useful for
-debugging.
+to build the docker image. The resulting image will have a `crosscon_hv` tag.
+After the image has been built you will need to create a container from that
+image, it can be done with this command:
 
-### Create the container from the image
+```bash
+docker create --name crosscon_hv_container crosscon_hv
+```
 
-`docker create --name crosscon_hv_container crosscon_hv`
+Then all that's left to do is run the image and enter the shell of the
+container, by running these commands:
 
-### Start the container
+```bash
+docker run -d --name crosscon_hv_container crosscon_hv tail -f /dev/null
+docker exec -it crosscon_hv_container /bin/bash
+```
 
-`docker run -d --name crosscon_hv_container crosscon_hv tail -f /dev/null`
+> Note: If at any point when rebuilding/rerunning the container you get error messages
+> similar to this one:
 
-`docker exec -it crosscon_hv_container /bin/bash`
+```bash
+docker: Error response from daemon: Conflict. The container name
+"/crosscon_hv_container" is already in use by container
+"d6ee75901fd0e090147d242c485651ebf5c4fc58e13d8363725a6cab830a9ba0". You have to
+remove (or rename) that container to be able to reuse that name.
+See 'docker run --help'.
+```
 
-### Remove the container
+Just use this command:
 
-If the container already exists, and you need to create it from newer
-image, use this command:
+```bash
+docker rm --force <hash_of_the_container_from_the_error_message>
+```
 
-`docker stop crosscon_hv_container`
+## Building the rpi4-ws demo
 
-`docker remove crosscon_hv_container`
+This section contains instructions on how to build and flash the rpi4-ws demo.
+After attaching to the container, it should look like this:
 
-After this you should be able to create a new container.
+```bash
+user in ~/CROSSCON-Hypervisor-and-TEE-Isolation-Demos/env λ docker exec -it crosscon_hv_container /bin/bash
+root@d6ee75901fd0:/work#
+```
 
-## After you attach
+Then you should `cd crosscon`, and follow the instructions from
+[the README](../rpi4-ws/README.md).
 
-1. Run `source env.sh` command. It will enable some of the tools that
-   were installed earlier to be found globally. It will also let you use
-   `gcc-9` as `gcc`.
+This will allow you to build the binaries, since the container has all the
+necessary dependencies, but here are some important tips to bear in mind:
 
-2. CROSSCON Hypervisor repository is already installed and it's
-   submodules were updated. It is available under the `crosscon`
-   directory.
+### Cross-compilator names
+
+Sometimes, the instructions are not 100% accurate when it comes to the
+`CROSS_COMPILE` flag in the `make` commands. The only time i have come across
```

Comment:
Please try to not use 1st person in documentation

Reply:
@tym2k1 fixed here: https://github.com/3mdeb/CROSSCON-Hypervisor-and-TEE-Isolation-Demos/pull/2/commits/79f038eeb55cfee0dff4ab66119618cda5efbf5f